crates/rpc/rpc-types/src/eth/pubsub.rs

```diff
@@ -14,6 +18,8 @@ pub enum SubscriptionResult {
     Log(Box<Log>),
     /// Transaction hash
     TransactionHash(H256),
+    /// Full Transaction
+    FullTransaction(Transaction),
```

Comment:
this variant won't be very common so we should box it as well to keep the enum size low
```suggestion
    FullTransaction(Box<Transaction>),
```